runtimes/core/src/api/manager.rs

```diff
@@ -436,3 +438,9 @@ fn listen_addr() -> String {
     }
     "0.0.0.0:8080".to_string()
 }
+
+#[derive(Debug)]
+pub struct CallOpts {
+    pub auth_data: Option<PValues>,
+    pub auth_user_id: Option<String>,
```

Comment:
Thoughts on making this into:

```suggestion
    pub auth: Option<(String, Option<PValues>)>,
```

Reply:
switched to a struct so the fields are named :eyes: